curvefs/src/client/fuse_client.h

```diff
@@ -279,6 +326,24 @@ class FuseClient {
     CURVEFS_ERROR UpdateParentInodeMCTimeAndInvalidNlink(
         fuse_ino_t parent, FsFileType type);
 
+    void BackGroundCmd();
+    void BackGroundCmdTask();
```

Comment:
Don't mix it all up in one class

Reply:
got it

Reply:
year, I will add todo, and then fix it later.